package/butterfly_angular/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: butterfly_angular
+description: A starting point for Dart libraries or applications.
+version: 0.0.1
+author: Jonah Williams <jonahwilliams@google.com>
+
+environment:
+  sdk: '>=1.20.1 <2.0.0'
+
+dependencies:
+ angular: '5.0.0-alpha'
+ butterfly:
+  path: '../butterfly'
```

Comment:
Hmm, wondering what this should be long-term. Probably keep it a `path` dependency and switch over to a versioned dependency in a custom script that publishes to pub?